tests/unit/utils.py

```diff
@@ -5,7 +5,7 @@
 import sys
 import os
 
-sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), "../listenstore"))
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), "../../listenstore"))
```

Comment:
This now causes a failure in lastfm tests:
```
______________________________________________ ERROR collecting db/tests/test_lastfm_user.py ______________________________________________
db/tests/test_lastfm_user.py:5: in <module>
    from tests.utils import generate_data
E   ImportError: No module named utils
```

Just checking... did you run tests as part of this PR?